versions/v6/functions.php

```diff
@@ -179,11 +179,11 @@ function display_photo_essay_item( $orientation, $item_id, $image_url, $title, $
 		case 'portrait':
 	?>
 		<div class="row">
-			<div class="img-col col-lg-7 offset-lg-0 col-md-10 offset-md-1 <?php if ( $alternate ) { ?>col-lg-push-5<?php } ?>">
+			<div class="img-col col-lg-7 offset-lg-0 col-md-10 offset-md-1 <?php if ( $alternate ) { ?>push-lg-5<?php } ?>">
 				<img class="photo-essay-img" src="<?php echo $image_url; ?>" alt="<?php echo $alt; ?>" title="<?php echo $title; ?>">
 				<div class="carat"></div>
 			</div>
-			<div class="caption-col col-lg-4 col-md-12 <?php if ( $alternate ) { ?>col-lg-pull-7 offset-lg-1<?php } else { ?>offset-lg-0<?php  } ?>">
+			<div class="caption-col col-lg-4 col-md-12 <?php if ( $alternate ) { ?>pull-lg-7 offset-lg-1<?php } else { ?>offset-lg-0<?php  } ?>">
```

Comment:
small beans, but to clean the classes up a bit: `col-md-12` classes are unnecessary here, it'll automatically take up 12cols when under the lg breakpoint since that breakpoint is defined.